src/dates/format-date.js

```diff
@@ -195,4 +195,32 @@ export default function formatDate(date, format, locale = "en") {
 
         return result;
     });
+}
+
+export function formatFormat(date, format, locale = "en") {
+    if (!isDate(date)) {
+        return format;
+    }
+
+    const info = localeInfo(locale);
+    const pattern = datePattern(format, info);
+
+    return pattern.replace(dateFormatRegExp, function(match) {
```

Comment:
Probably I am missing something here, but isn't the `pattern` the full format that we need?